examples/arm/setup.sh

```diff
@@ -86,6 +86,11 @@ fi
 ethos_u_repo_url="https://review.mlplatform.org/ml/ethos-u/ethos-u"
 ethos_u_base_rev="24.08"
 
+# tosa reference model
+# https://review.mlplatform.org/c/tosa/reference_model/+/12409
```

Comment:
I'd remove this link; someone's going to update the model_rev and not update this URL

Reply:
removed.